### DISCUSSION TITLE
I want to build radix-vue locally for open source contribution

### DISCUSSION THREAD

**Discussion by ThejanNim** [unovue/reka-ui#706] [Q&A] [answered]
Can anyone help to me run and build radix-vue in my local machine?

**Comment by zernonia** [comment]
This should works

1. Clone the repo
2. Run `pnpm i`
3. Run `pnpm build` to run build `radix-vue` locally

**Reply by ThejanNim** [reply]
Thanks for your quick reply, how to install it to my project without using npm i radix-vue? I want to change radix-vue locally and install it my personal project.

**Reply by zernonia** [reply, marked as answer]
You can use `npm link`. Can check this guide out https://medium.com/@AidThompsin/how-to-npm-link-to-a-local-version-of-your-dependency-84e82126667a

**Reply by ThejanNim** [reply]
Thanks for your reply
